src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
+
+namespace NUnit.VisualStudio.TestAdapter
+{
+    public class AdapterSettings
+    {
+        private XmlNode _runConfiguration;
+        private XmlNode _testRunParameters;
+        private XmlNode _nunitAdapter;
+
+        #region Properties
+
+        public string MaxCpuCount
+        {
+            get { return GetInnerText(_runConfiguration, "MaxCpuCount"); }
+        }
+
+        public string ResultsDirectory
+        {
+            get { return GetInnerText(_runConfiguration, "ResultsDirectory"); }
+        }
```

Comment:
So if this is set to 2, for example, will VS start two different execution processes and feed different assemblies to them? If that's so, then maybe we never have to implement parallel assembly execution in the adapter itself.

The setting we currently support is NumberOfTestWorkers, which controls the threads used by the framework. The user would need to adjust that based on the number of NUnit assemblies being run.

If you have any suggestions for how to set up a test bed for this that would be helpful.